src/core/client/ui/components/ValidationMessage/ValidationMessage.css

```diff
@@ -0,0 +1,35 @@
+.root {
+  composes: validationMessage from "talk-ui/shared/typography.css";
+  position: relative;
+  display: inline-flex;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 7px 10px;
```

Comment:
`padding: calc(0.5 * var(--spacing-unit)) var(--spacing-unit)`

Reply:
Wow. That line is ugly. Maybe we can add some extra func here. Adding the line 👍 .